frontend/src/hooks/useChatSession.ts

```
@@ -0,0 +1,359 @@
+
+      if (modelSupportsVision && images && images.length > 0) {
+        const parts: ChatContentPart[] = [{ type: "text", text: content }];
+        for (const file of images) {
+          try {
+            const url = await fileToDataURL(file);
+            parts.push({ type: "image_url", image_url: { url } });
+          } catch (error) {
+            console.error("[useChatSession] Failed to convert image to data URL:", error);
+            // Continue with other images instead of failing the entire operation
+            // TODO: Consider surfacing this error to the user in the future
+            continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)